robottelo/ui/base.py

```diff
@@ -300,6 +299,25 @@ def wait_until_element(self, locator, timeout=12, poll_frequency=0.5):
             )
             return None
 
+    def wait_until_element_go(self, locator, timeout=12, poll_frequency=0.5):
```

Comment:
What is a difference between this method and `wait_until_element_is_not_visible`